lib/matplotlib/tests/test_artist.py

```diff
@@ -562,3 +562,8 @@ def draw(self, renderer, extra):
 
     assert 'aardvark' == art.draw(renderer, 'aardvark')
     assert 'aardvark' == art.draw(renderer, extra='aardvark')
+
+
+def test_artist_seed_update():
+    martist.Artist().set_sketch_params(seed=2345)
+    assert mpl.rcParams['path.sketch_seed'] == 2345
```

Comment:
Have changed it , passing a seed now changes the private variable self._sketch_seed and the value for seed is not (seed or self._sketch_seed) where sketch_seed is initialized with rcParam, so that something like :
```
rcParams['path.sketch_seed'] =200
ln.set_sketch_params(20,30,40)
```
uses 200 as seed rather than 0.